src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Badging for app icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-10-08
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
+alt: Phone showing several notification badges
```

Comment:
```suggestion
alt: Phone showing several notification badges
draft: true
```

@petele you mentioned that you'd prefer to publish all the posts at once. When the time comes we can submit a follow-up PR to remove this flag.